components/tidb_query/src/rpn_expr/impl_math.rs

```diff
@@ -1220,4 +1252,71 @@ mod tests {
             assert_eq!(got, e);
         }
     }
+
+    #[test]
+    fn test_round_real() {
+        let test_cases = vec![
```

Comment:
Looks like you can write None case here directly.

Reply:
refactored.